pylint/checkers/classes/class_checker.py

```diff
@@ -1776,10 +1777,16 @@ def _check_protected_attribute_access(
             Klass.
         """
         attrname = node.attrname
```

Comment:
```suggestion
        attrname = node.attrname
        if not is_attr_protected(attrname):
            return
```
Then we can de-indent everything after that.